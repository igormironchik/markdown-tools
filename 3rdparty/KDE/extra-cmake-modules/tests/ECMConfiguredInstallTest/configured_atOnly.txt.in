[General]
Foo=@FOO@
Bar=${BAR}
