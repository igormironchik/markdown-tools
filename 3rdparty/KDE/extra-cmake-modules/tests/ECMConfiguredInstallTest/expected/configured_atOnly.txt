[General]
Foo=myFoo
Bar=${BAR}
